client/karma.js

```diff
@@ -112,7 +112,12 @@ var Karma = function (socket, iframe, opener, navigator, location) {
     if (url) {
       message = msg + '\nat ' + url + (line ? ':' + line : '')
     }
-
+    
+    // create an object with the string representation of the message to ensure all its content is properly 
+    // transferred to the console log
+    message = {message: message, str: message.toString().replace(/\t/g, "    ").split("\n")}
```

Comment:
the variable type is not changed, the message already is an object that will later on be converted to a string with JSON.stringify()